packages/docs/components/atoms/Figure/twig-api.md

```diff
@@ -76,6 +76,30 @@ Use absolute position on the image holder instead of relative.
 
 Wether to enable the display of the figure inline or not. When `inline`, the root element will have a max-width set corresponding to the `width` given. Use with caution.
 
+### `placeholder`
+
+- Type: `string`
+
+Use a custom placeholder instead of the generic placeholder :
```

Comment:
**chore:** There is no space before a `:` in English

```suggestion
Use a custom placeholder instead of the generic placeholder:
```

Reply:
Pardon my french